bench/commands/utils.py

```diff
@@ -8,11 +8,12 @@
 
 @click.command('start', help="Start Frappe development processes")
 @click.option('--no-dev', is_flag=True, default=False)
+@click.option('--no-prefix', is_flag=True, default=False)
```

Comment:
Can you add a description for this flag. Might be unclear what this is supposed to do from the CLI

Reply:
sure

Reply:
Where should I add the description? Can you please guide me.

Reply:
`@click.option` accepts a `help` parameter where you can add it.

Reply:
I was assuming that only, just wanted to make sure.
Thanks @gavindsouza

Reply:
Can you please check now. I have added.